keras_cv/utils/bounding_box_test.py

```diff
@@ -114,3 +118,18 @@ def test_bounding_box_padding(self):
             ValueError, "Target shape should be larger than bounding box shape"
         ):
             bounding_box.pad_bounding_box_batch_to_shape(bounding_boxes, target_shape)
+
+    def test_mask_to_bbox(self):
+        target_boundingbox = [1, 1, 1, 1]
```

Comment:
can we try a bouding box where the coordinates are not all 1s?


[1, 1, 5, 5] maybe?

Reply:
@LukeWood make sense,
btw should be check top, left as well, like the sequence of them
using 

1,2, 3,4 may be?
in all test cases